make/e2e-setup.mk

```diff
@@ -43,7 +43,8 @@ IMAGE_sampleexternalissuer_arm64 := ghcr.io/cert-manager/sample-external-issuer/
 IMAGE_projectcontour_arm64 := ghcr.io/projectcontour/contour:v1.29.1@sha256:dbfec77951e123bf383a09412a51df218b716aaf3fe7b2778bb2f208ac495dc5
 
 # https://github.com/letsencrypt/pebble/releases
-PEBBLE_COMMIT = ba5f81dd80fa870cbc19326f2d5a46f45f0b5ee3
+# v2.7.0
+PEBBLE_COMMIT = bbe7775c27078417e379287a431fa5a716a6788d
```

Comment:
This is the first release with support for ACME profiles:
 * https://github.com/letsencrypt/pebble/releases/tag/v2.7.0